flair/values.yaml

```diff
@@ -0,0 +1,31 @@
+replicaCount: 1
+
+image:
+  repository: smartcitiesdata/flair
+  tag: development
+  pullPolicy: Always
+
+localstack:
+  enabled: false
+
+kafka:
+  #comma delimited list of kafka seed brokers
+  brokers: data-broker:9092
+  strimziCluster: streaming-service # http://strimzi.io/docs/0.4.0/#_topic_configmap
+
+nameOverride: ""
+fullnameOverride: ""
+nodeSelector: {}
+tolerations: []
+affinity: {}
+
+flair:
+  dlq_topic: streaming-dead-letters
+  data_topic: streaming-persisted
```

Comment:
maaaaybe this should just be `persisted` since that's more generic, but I think this is basically just as good.